scripts/west/zap_gui.py

```diff
@@ -31,33 +33,52 @@ def do_add_parser(self, parser_adder):
                                          description=self.description)
         parser.add_argument('-z', '--zap-file', type=existing_file_path,
                             help='Path to data model configuration file (*.zap)')
-        parser.add_argument('-j', '--zcl-json', type=existing_file_path,
-                            help='Path to data model definition file (zcl.json)')
+        parser.add_argument('-j', '--zcl-json', type=str,
+                            help='Path to data model definition file (zcl.json). If new clusters are added using --clusters, the new zcl.json file will be be created and used.')
```

Comment:
It will be created if there isn't one, right? Otherwise the existing zcl.json will be updated.
```suggestion
                            help='Path to data model definition file (zcl.json). If new clusters are added using --clusters, the new zcl.json file will be created and used.')
```